### DISCUSSION TITLE
Schwarzenbach Schöggi

### DISCUSSION THREAD

**Discussion by ghost** [Estheryu991/SwissGerman_Dictionary#79] [General]
[Schwarzenbach Schöggi](https://www.schwarzenbach.ch/produkt-kategorie/geschenke/geschenksets/)

**Comment by GoogleNaturalLanguage** [comment]
Schwarzenberg Schokolade 
[mint](https://www.schwarzenbach.ch/produkt-kategorie/schokolade/tafelschokoladen/?_produzent=chocolate-and-love)

**Reply by GoogleNaturalLanguage** [reply]
![E87470AC-401D-46A1-8E58-4B8DB8E31205](https://user-images.githubusercontent.com/118141056/201550622-3dc93b94-4a2b-4eaf-8d48-4726d780e208.jpeg)
